Notebooks/Simulations/skypy_lenstronomy.ipynb

```diff
@@ -29,7 +29,7 @@
      "name": "stdout",
```

Comment:
<p>Can you update the versions of SkyPy and lenstronomy for this notebook, and also add your name to the Authors ;)</p><p> </p><p>Can you also add a link to the SkyPy examples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>